migrations/0043_normalize_claim_creds.up.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE public.claim_creds ADD COLUMN created_at timestamp without time zone;
+ALTER TABLE public.claim_creds ADD COLUMN updated_at timestamp without time zone;
+
+create view normalized_claim_creds as (
+  select
+    claim_id,
+    blinded_creds,
+    signed_creds,
+    json_array_length(signed_creds_length),
+    batch_proof,
+    public_key,
+    issuer_id,
+    COALESCE(claim_creds.created_at, '2021-12-01'::date::timestamp without time zone) AS created_at,
+    COALESCE(claim_creds.updated_at, '2021-12-01'::date::timestamp without time zone) AS updated_at,
+  from claim_creds
```

Comment:
Maybe we use the current date or one day in the future? That way we can expect very record to be created at or before the timestamp rather than at, before or after

Reply:
are you suggesting to set updated_at to current date in case of null?